common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -394,6 +394,7 @@ def get_html(self):
             'progress_status': Progress.to_js_status_str(progress),
             'progress_detail': Progress.to_js_detail_str(progress),
             'content': self.get_problem_html(encapsulate=False),
+            'graded': self.graded,
```

Comment:
I think it would be good to have an end-to-end test (bok choy) that verifies that a problem in a graded unit shows "graded", whereas one in a non-graded unit says "ungraded". There are tests of the individual parts, but nothing that makes sure that this line is actually correct (as far as I can tell).

We will have to figure out where to put such a test. Perhaps in a test class that @staubina is creating for the focus tests? Let's talk about it next week.